### ISSUE TITLE
get all tests to run on BCDB [development_bcdb_sqlite]

### ISSUE THREAD

**Issue by despiegk** [threefoldtech/jumpscaleX_core#51]

**Comment by BolaNasr** [comment]
all Tests works good except ( 4_models)

**Comment by hossnys** [comment]
as branch development_bcdb_sqlite merged , so when running BCDB tests on development this error found :
```
[b'0', [b'{"llist2": [], "name": "somename1", "email": "", "nr": 1, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 1}', b'{"llist2": [], "name": "somename2", "email": "", "nr": 2
, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 2}', b'{"llist2": [], "name": "somename3", "email": "", "nr": 3, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist
": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 3}', b'{"llist2": [], "name": "somename4", "email": "", "nr": 4, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id":
 4}', b'{"llist2": [], "name": "somename5", "email": "", "nr": 5, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 5}', b'{"llist2": [], "name": "somename6", "email": "", "nr": 6, 
"date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 6}', b'{"llist2": [], "name": "somename7", "email": "", "nr": 7, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist":
 [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 7}', b'{"llist2": [], "name": "somename8", "email": "", "nr": 8, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 8
}', b'{"llist2": [], "name": "somename9", "email": "", "nr": 9, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 9}', b'{"llist2": [], "name": "somename10", "email": "", "nr": 10, 
"date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 10}']]
1
b'{"llist2": [], "name": "somename2", "email": "", "nr": 2, "date_start": 0, "description": "", "cost_estimate": 0.0, "llist": [], "llist3": [1.0, 2.0, 3.0], "llist4": ["1", "2", "3"], "id": 2}'
--TRACEBACK------------------
<stdin> in <module>
    1    
/sandbox/lib/jumpscale/Jumpscale/data/bcdb/BCDBFactory.py in test
    484    self._test_run(name=name)
/sandbox/lib/jumpscale/Jumpscale/core/BASECLASSES/TestTools.py in _test_run
    53    res = self.__test_run(name=name, obj_key=obj_key, die=die, **kwargs)
/sandbox/lib/jumpscale/Jumpscale/core/BASECLASSES/TestTools.py in __test_run
    165    self.__test_run(name=name, obj_key=obj_key, **kwargs)
/sandbox/lib/jumpscale/Jumpscale/core/BASECLASSES/TestTools.py in __test_run
    154    self._code_run(name=name, path=path, obj_key=obj_key, die=die, **kwargs)
/sandbox/lib/jumpscale/Jumpscale/core/BASECLASSES/TestTools.py in _code_run
    75    res = method(self=self, **kwargs)
/sandbox/lib/jumpscale/Jumpscale/data/bcdb/tests/8_keys.py in main
    115    assert o.nr2 == 2147483647
-----------------------------
Sun 15 14:41:46 bcdb/tests/8_keys.py - 115 - main                               : EXCEPTION: 
    AssertionError()
```

**Comment by BolaNasr** [comment]
@hossnys  fixed at 16fbd6bf057dc7aec8a5f1984037bb77c47de26d

**Comment by hossnys** [comment]
verified on development branch and tests are **ALL OK** 

```
[]
['data', 'schemas', 'info']
namespaces:['1']
urls:['jumpscale.bcdb.user.2', 'jumpscale.bcdb.circle.2', 'jumpscale.bcdb.acl.2', 'jumpscale.bcdb.acl.circle.2', 'jumpscale.bcdb.acl.user.2', 'jumpscale.bcdb.meta.namespace.2', 'jumps
cale.rdb.client', 'despiegk.test', 'jumpscale.currencylayer.client', 'jumpscale.zdb.server.1', 'jumpscale.startupcmd.1', 'jumpscale.zdb.client.1', 'despiegk.test.sqlite', 'jumpscale.s
onic.server.1', 'jumpscale.sonic.client', 'jumpscale.schema.test.a', 'threefoldtoken.order.buy', 'jumpscale.bcdb.test.room', 'threefoldtoken.order.sell', 'jumpscale.bcdb.test.house', 
'threefoldtoken.wallet', 'despiegk.test5.acl', 'despiegk.test2', 'threefoldtoken.wallet.test', 'threefoldtoken.wallet.test2', 'test.schema.1', 'jsx.master.1', 'jsx.subschema.1', 'jsx.
subschema.2', 'jsx.subschema.3', 'jsx.subschema.3.1', 'jsx.subschema.3.1.1', 'jsx.subschema.3.1.2', 'jsx.subschema.3.2.1', 'jsx.subschema.3.2.2', 'jsx.subschema.3.2']
['jumpscale.bcdb.user.2', 'jumpscale.bcdb.circle.2', 'jumpscale.bcdb.acl.2', 'jumpscale.bcdb.acl.circle.2', 'jumpscale.bcdb.acl.user.2', 'jumpscale.bcdb.meta.namespace.2', 'jumpscale.
rdb.client', 'despiegk.test', 'jumpscale.currencylayer.client', 'jumpscale.zdb.server.1', 'jumpscale.startupcmd.1', 'jumpscale.zdb.client.1', 'despiegk.test.sqlite', 'jumpscale.sonic.
server.1', 'jumpscale.sonic.client', 'jumpscale.schema.test.a', 'threefoldtoken.order.buy', 'jumpscale.bcdb.test.room', 'threefoldtoken.order.sell', 'jumpscale.bcdb.test.house', 'thre
efoldtoken.wallet', 'despiegk.test5.acl', 'despiegk.test2', 'threefoldtoken.wallet.test', 'threefoldtoken.wallet.test2', 'test.schema.1', 'jsx.master.1', 'jsx.subschema.1', 'jsx.subsc
hema.2', 'jsx.subschema.3', 'jsx.subschema.3.1', 'jsx.subschema.3.1.1', 'jsx.subschema.3.1.2', 'jsx.subschema.3.2.1', 'jsx.subschema.3.2.2', 'jsx.subschema.3.2']
comparing key:testvfs_data_1_threefoldtoken.wallet.test_3 with key:testvfs_data_1_threefoldtoken.wallet.test_3
Dav server running on port 4444
TEST OK
'OK'

```
